src/service-worker.js

```diff
@@ -0,0 +1,76 @@
+const SERVICE_WORKER_VERSION = 1;
+const CACHE_NAME = `comma-connect-cache-v${SERVICE_WORKER_VERSION}`;
+
```

Comment:
Incrementing the version number forces browsers to update their cached service worker when you deploy changes - without it, users might keep running old versions.